packages/utils/src/date/index.ts

```diff
@@ -8,6 +8,10 @@ interface ConversionOptions {
 	sourceTimeZone?: string,
 }
 
+interface Map {
```

Comment:
Maybe you'll want to be more specific on this
```suggestion
interface DateFormatOptions {
```